sdk/resourcemanager/azure-resourcemanager-samples/pom.xml

```diff
@@ -0,0 +1,206 @@
+      <activation>
+        <property>
+          <name>!maven.test.skip</name>
+        </property>
+      </activation>
+      <dependencies>
+        <dependency>
+          <groupId>com.azure.resourcemanager</groupId>
+          <artifactId>azure-resourcemanager-resources</artifactId>
+          <version>2.0.0-beta.4</version> <!-- {x-version-update;com.azure.resourcemanager:azure-resourcemanager-resources;current} -->
+          <type>test-jar</type>
+          <scope>test</scope>
+        </dependency>
+      </dependencies>
+    </profile>
+  </profiles>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>3.0.0-M3</version> <!-- {x-version-update;org.apache.maven.plugins:maven-enforcer-plugin;external_dependency} -->
+        <configuration>
+          <rules>
+            <bannedDependencies>
+              <includes>
+                <include>org.apache.httpcomponents:httpclient:[4.5.12]</include> <!-- {x-include-update;resourcemanager_org.apache.httpcomponents:httpclient;external_dependency} -->
+                <include>io.fabric8:kubernetes-client:[4.9.1]</include> <!-- {x-include-update;io.fabric8:kubernetes-client;external_dependency} -->
+                <include>com.jcraft:jsch:[0.1.55]</include> <!-- {x-include-update;resourcemanager_com.jcraft:jsch;external_dependency} -->
+                <include>org.slf4j:slf4j-simple:[1.7.25]</include> <!-- {x-include-update;org.slf4j:slf4j-simple;external_dependency} -->
+                <include>com.google.guava:guava:[28.1-jre]</include> <!-- {x-include-update;resourcemanager_com.google.guava:guava;external_dependency} -->
+                <include>com.github.docker-java:docker-java:[3.2.1]</include> <!-- {x-include-update;com.github.docker-java:docker-java;external_dependency} -->
+                <include>com.microsoft.sqlserver:mssql-jdbc:[6.4.0.jre7]</include> <!-- {x-include-update;com.microsoft.sqlserver:mssql-jdbc;external_dependency} -->
+                <include>org.eclipse.jgit:org.eclipse.jgit:[4.5.7.201904151645-r]</include> <!-- {x-include-update;org.eclipse.jgit:org.eclipse.jgit;external_dependency} -->
+                <include>com.github.cverges.expect4j:expect4j:[1.6]</include> <!-- {x-include-update;com.github.cverges.expect4j:expect4j;external_dependency} -->
+                <include>commons-net:commons-net:[3.6]</include> <!-- {x-include-update;commons-net:commons-net;external_dependency} -->
+              </includes>
+            </bannedDependencies>
+          </rules>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.jacoco</groupId>
+        <artifactId>jacoco-maven-plugin</artifactId>
+        <version>0.8.5</version> <!-- {x-version-update;org.jacoco:jacoco-maven-plugin;external_dependency} -->
+        <configuration>
+          <excludes>
+            <exclude>com/azure/resourcemanager/**/*</exclude>
```

Comment:
Is this `exclude` correct?

Reply:
Do we need to run Jacoco in samples?

Reply:
If we want to disable jacoco, is there a clearer approach?

Something like
```
<configuration>
      <skip>true</skip>
</configuration>
```

Reply:
Removed.